src/audio/drc/drc_generic.c

```diff
@@ -8,41 +8,46 @@
 #include <sof/audio/drc/drc.h>
 #include <sof/audio/drc/drc_math.h>
 #include <sof/audio/format.h>
+#include <sof/math/decibels.h>
 #include <sof/math/numbers.h>
 #include <stdint.h>
 
+#define ONE_Q20        Q_CONVERT_FLOAT(1.0f, 20)                /* Q12.20 */
```

Comment:
I would prefer we don't do this with the defines, I don't think we need to name these values. Keep them inline please (unless anyone objects)